static/monorail-2.0.yaml

```diff
@@ -60,6 +60,23 @@ definitions:
         type: string
       username:
         type: string
+  hooks.2.0_HooksBase:
```

Comment:
@brianparry : I need your confirm here. Since we have created a schema file (schemas/2.0/hooks.2.0.json), so whether this inline schema is still needed?